hello.go

```diff
@@ -3,11 +3,11 @@ package main
 import "fmt"
 
 func main() {
-    SayHello()
+	SayHello2()
 }
 
-func SayHello() {
- fmt.Println("Hello world!!") // gofmt test
-//    return
-//    fmt.Println("Unreachable line") // got vet error test
+func SayHello2() {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function SayHello2 should have comment or be unexported